dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/utils/StAXUtils.java

```diff
@@ -20,24 +22,115 @@ public class StAXUtils {
 
     private static final TraceComponent tc = Tr.register(StAXUtils.class);
 
-    public static final String IBM_XLXP2_XML_OUTPUT_FACTORY = "com.ibm.xml.xlxp2.api.stax.XMLOutputFactoryImpl";
+    // The JAXB Validator API was dropped in EE10, so we can check that API to know if we're on EE10 or above
+    private static Boolean isEE10 = false;
```

Comment:
```suggestion
    private static boolean isEE10 = false;
```

No reason for it to be a primitive wrapper since a null state is not one of the options in the logic.